.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in The Power of Us Hub.
```

Comment:
@mrrigney I didn't see this one. Line 1: For support questions, please use the K-12 ([Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE).

Reply:
@aarsvoboda I modified this line, can you please verify?  Made you an approver

Reply:
@mrrigney Here is my suggestion for Line 1:
 **Note: FOR INTERNAL USE ONLY. For support questions, please use the [Sector: K-12 Education](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE) group in Trailblazer Community.

Reply:
@aarsvoboda I made this change